src/TextField/TextField.jsx

```diff
@@ -289,12 +279,15 @@ const TextField = React.createClass({
     ) : null;
 
     let floatingLabelTextElement = floatingLabelText ? (
-      <label
-        style={this.prepareStyles(styles.floatingLabel, this.props.floatingLabelStyle)}
+      <TextFieldLabel
+        muiTheme={this.state.muiTheme}
+        style={this.mergeStyles(styles.floatingLabel, this.props.floatingLabelStyle)}
```

Comment:
Should we consider extracting the rest of the styles from `styles.floatingLabel` and putting it in the `TextFieldLabel` component? That way the only thing `TextField` has to pass is the `floatingLabelStyle` prop?